charts/cluster-addons/templates/monitoring/loki-stack.yaml

```diff
@@ -1,86 +1,95 @@
-{{- if and .Values.monitoring.enabled .Values.monitoring.lokiStack.enabled }}
+{{- if and .Values.monitoring.enabled .Values.monitoring.loki.enabled }}
 ---
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ include "cluster-addons.componentName" (list . "loki-stack") }}-config
+  name: {{ include "cluster-addons.componentName" (list . "loki") }}-config
   labels:
-    {{- include "cluster-addons.componentLabels" (list . "loki-stack") | nindent 4 }}
+    {{- include "cluster-addons.componentLabels" (list . "loki") | nindent 4 }}
     addons.stackhpc.com/watch: ""
 stringData:
   defaults: |
     loki:
       serviceMonitor:
         enabled: true
+  overrides: |
+    {{- toYaml .Values.monitoring.loki.release.values | nindent 4 }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "cluster-addons.componentName" (list . "promtail") }}-config
+  labels:
+    {{- include "cluster-addons.componentLabels" (list . "promtail") | nindent 4 }}
+    addons.stackhpc.com/watch: ""
+stringData:
+  defaults: |
     promtail:
-      serviceMonitor:
-        enabled: true
-      # Get promtail to scrape systemd services
       config:
-        snippets:
-          extraScrapeConfigs: |
-            - job_name: journal
-              journal:
-                path: /var/log/journal
-                max_age: 12h
-                labels:
-                  job: systemd-journal
-              relabel_configs:
-                - source_labels: ['__journal__systemd_unit']
-                  target_label: 'unit'
-                - source_labels: ['__journal__hostname']
-                  target_label: 'hostname'
-                - source_labels: ['__journal_priority_keyword']
-                  target_label: level
-      # Mount journal directory and machine-id file into promtail pods
-      extraVolumes:
-        - name: journal
-          hostPath:
-            path: /var/log/journal
-        - name: machine-id
-          hostPath:
-            path: /etc/machine-id
-      extraVolumeMounts:
-        - name: journal
-          mountPath: /var/log/journal
-          readOnly: true
-        - name: machine-id
-          mountPath: /etc/machine-id
-          readOnly: true
-    grafana:
-      sidecar:
-        datasources:
-          enabled: false
+        serviceMonitor:
+          enabled: true
+        # Get promtail to scrape systemd services
+        config:
+          snippets:
```

Comment:
It looks like we've got two layers of `config` key here now - is this correct? The values now look like
```yaml
promtail:
  config:
    config:
      snippets:
        ...
```